_rules/audio-transcript-2eb176.md

```diff
@@ -131,7 +154,7 @@ This `audio` element has native player controls and an incorrect external transc
 
 #### Failed Example 4
 
-This `audio` element has an `autoplay` attribute and an incorrect external transcript.
+This `audio` element has an [autoplay][] attribute and an incorrect external transcript.
```

Comment:
Since autoplay here is referring to the HTML attribute I don't think it needs to link to the definition.